crates/aptos-inspection-service/src/server/profiling.rs

```diff
@@ -16,116 +16,58 @@ pub fn handle_profiling_request() -> (StatusCode, Body, String) {
     let mut contents = String::new();
     file.read_to_string(&mut contents)
         .expect("Failed to read file");
-    (
-        StatusCode::OK,
-        Body::from(contents),
-        CONTENT_TYPE_HTML.into(),
-    )
+
+    (StatusCode::OK, Body::from(contents), CONTENT_TYPE_HTML.into(),)
 }
 
-//TODO: use aptos-profiler crater
 pub fn handle_cpu_profiling_request() -> (StatusCode, Body, String) {
-    let guard = pprof::ProfilerGuard::new(100).unwrap();
-    let five_secs = time::Duration::from_millis(5000);
-    thread::sleep(five_secs);
-
-    if let Ok(report) = guard.report().build() {
-        let file = File::create("./flamegraph.svg").unwrap();
-        report.flamegraph(file).unwrap();
-
-        println!("report: {:?}", &report);
-    };
+    //Call aptos-profiling cpu profiling 
+    let config = ProfilerConfig::new_with_defaults();
+    let handler = ProfilerHandler::new(config);
+    let cpu_profiler = handler.get_cpu_profiler();
+    cpu_profiler.start_profiling();
```

Comment:
if the start_profiling returns error, we should return a different status code? similarly for the rest of functions